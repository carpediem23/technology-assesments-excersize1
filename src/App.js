import React from 'react';
import './App.scss';

const App = () => {
  return (
    <div className="app-container">
      <p>react app</p>
    </div>
  );
};

export default App;
