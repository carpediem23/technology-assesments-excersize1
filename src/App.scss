@import 'bootstrap/scss/bootstrap-utilities';
@import 'bootstrap/scss/bootstrap-reboot';
@import 'bootstrap/scss/bootstrap-grid';



$body-background-color: rgba(255, 255, 255, 1);
$body-background--gradident-color: rgba(236, 236, 236, 1);

body {
  background: radial-gradient(
    circle,
    $body-background-color 30%,
    $body-background--gradident-color 100%
  );

  .app-container {
    margin-top: 1em;
    padding: 1em;
    box-shadow: 3px 3px 6px -3px #000000;
    background-color: #fff;
  }
}
