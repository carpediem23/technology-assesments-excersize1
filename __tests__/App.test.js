import React from 'react';
import renderer from 'react-test-renderer';
import App from '../src/App';

test('App rendering', () => {
  const component = renderer.create(<App />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
