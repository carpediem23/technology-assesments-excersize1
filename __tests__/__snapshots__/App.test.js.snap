// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App rendering 1`] = `
<main
  className="container"
>
  <div
    className="AddressBook shadow rounded"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        <form
          className="AdressBook-search-form"
          name="AdressBook-search-form"
        >
          <div
            className="AdressBook-search-form-bar"
          >
            <i
              className="AdressBook-search-form-icon fa fa-search"
            />
            <input
              aria-describedby="emailHelp"
              className="AdressBook-search-form-input"
              id="search-term"
              onChange={[Function]}
              placeholder="SEARCH (Client Name / Policy Number)"
              type="text"
            />
          </div>
        </form>
      </div>
    </div>
  </div>
</main>
`;
